lib/ret/hub.ex

```diff
@@ -142,13 +151,46 @@ defmodule Ret.Hub do
     end
   end
 
+  def get_scene_or_scene_listing_by_id(nil) do
+    SceneListing.get_random_default_scene_listing()
+  end
+
+  def get_scene_or_scene_listing_by_id(id) do
+    Scene.scene_or_scene_listing_by_sid(id)
+  end
```

Comment:
I believe these can be made `defp`, and we might as well be consistent with `sid` in the function name, since it's for internal use anyway.